NETPROGRAM/String_Exercise/Exercise08.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise08
+    {
+
+        public string RemoveCharRedundancy(string s)
+        {
+            var stringBuilder = new StringBuilder();
+            stringBuilder.Append(s[0]);
+
+            for (var i = 1; i < s.Length; i++)
+            {
+                if (stringBuilder.ToString().Last() != s[i])
```

Comment:
Thao tác `ToString()` sẽ chuyển toàn bộ string builder về string và việc đó thực hiện trong vòng lặp. Không tốt tí nào

```suggestion
                if (sb[sb.Length - 1] != s[i])
```

Hoặc đơn giản hơn là dùng 1 biến kiểu char và lưu lại ký tự gần nhất

Reply:
done